SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -307,6 +308,7 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	StaticComponentView = NewObject<USpatialStaticComponentView>();
 	SnapshotManager = NewObject<USnapshotManager>();
 	SpatialMetrics = NewObject<USpatialMetrics>();
+	VirtualWorkerTranslator = new SpatialVirtualWorkerTranslator();
```

Comment:
Do we have a standard way we deal with non-actor/UObject/Unreal memory management?

Reply:
TUniquePtr if it has to be heap allocated. Otherwise just an member variable.

Reply:
The style guide is avoid raw new and delete. SharedPtr is currently fairly overused in the repo.

Reply:
Updated this now.